integration/oqs_openssl/run.sh

```diff
@@ -52,31 +52,46 @@ echo "Cloning openssl"
 if [ ! -d "${BASEDIR}/openssl" ] ; then
     git clone https://github.com/open-quantum-safe/openssl "${BASEDIR}/openssl" >> $LOGS 2>&1
 fi
+
 cd "${BASEDIR}/openssl"
 git clean -d -f -x >> $LOGS 2>&1
 git checkout -- . >> $LOGS 2>&1
 
-rm -rf "${BASEDIR}/openssl-1_0_2-master"
-cp -pr "${BASEDIR}/openssl" "${BASEDIR}/openssl-1_0_2-master"
-cd "${BASEDIR}/openssl-1_0_2-master"
+if [ ! -d "${BASEDIR}/openssl_1_1_1-stable" ] ; then
+	git clone -b OpenSSL_1_1_1-stable https://github.com/openssl/openssl.git "${BASEDIR}/openssl_1_1_1-stable" >> $LOGS 2>&1 
+fi
+
+cd "${BASEDIR}/openssl_1_1_1-stable"
+case "$OSTYPE" in
+  darwin*) CFLAGS=-fPIC  ./Configure shared --prefix="${BASEDIR}/install-openssl_1_1_1-stable" darwin64-x86_64-cc >> $LOGS 2>&1 ;;
```

Comment:
The change from `./Configure no-shared` to `CFLAGS=-fPIC ./Configure shared` is causing a bunch of problems on my Mac.  Is there a reason you made this change?

Reply:
No, I think it's due to an out-of-sync commit. As noted in my comment above; some changes was applied to run.sh (I'm not sure why and how), but I've only added the windows test file.